<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Data base is connected]]>
    </line>
    <line>
      <![CDATA[Chrome is connected]]>
    </line>
    <line>
      <![CDATA[Logged in as Admin.....]]>
    </line>
    <line>
      <![CDATA[Loggiing out ....]]>
    </line>
    <line>
      <![CDATA[Logged in as Admin.....]]>
    </line>
    <line>
      <![CDATA[Loggiing out ....]]>
    </line>
    <line>
      <![CDATA[Logged in as Admin.....]]>
    </line>
    <line>
      <![CDATA[Loggiing out ....]]>
    </line>
    <line>
      <![CDATA[Logged in as Admin.....]]>
    </line>
    <line>
      <![CDATA[Loggiing out ....]]>
    </line>
    <line>
      <![CDATA[Browser is closing...]]>
    </line>
    <line>
      <![CDATA[Data base is closing...]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-14T15:27:50 IST" name="Default suite" finished-at="2023-07-14T15:27:58 IST" duration-ms="7781">
    <groups>
      <group name="regressionTest">
        <method signature="ReportsPageTest.reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" name="reportsTest" class="pgs.hms.test.ReportsPageTest"/>
      </group> <!-- regressionTest -->
      <group name="smokeTest">
        <method signature="ReportsPageTest.reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" name="reportsTest" class="pgs.hms.test.ReportsPageTest"/>
      </group> <!-- smokeTest -->
    </groups>
    <test started-at="2023-07-14T15:27:50 IST" name="Default test" finished-at="2023-07-14T15:27:58 IST" duration-ms="7781">
      <class name="pgs.hms.test.ReportsPageTest">
        <test-method is-config="true" signature="connectToDB()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:50 IST" name="connectToDB" finished-at="2023-07-14T15:27:50 IST" duration-ms="401" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Data base is connected]]>
            </line>
          </reporter-output>
        </test-method> <!-- connectToDB -->
        <test-method is-config="true" signature="setUpBrowser()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:50 IST" name="setUpBrowser" finished-at="2023-07-14T15:27:53 IST" duration-ms="2216" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Chrome is connected]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpBrowser -->
        <test-method is-config="true" signature="loginToApplicationAsAdmin()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:53 IST" name="loginToApplicationAsAdmin" finished-at="2023-07-14T15:27:53 IST" duration-ms="785" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Logged in as Admin.....]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToApplicationAsAdmin -->
        <test-method retried="true" signature="reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:53 IST" name="reportsTest" finished-at="2023-07-14T15:27:54 IST" duration-ms="729" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at pgs.hms.test.ReportsPageTest.reportsTest(ReportsPageTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportsTest -->
        <test-method is-config="true" signature="logoutToApplication()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:54 IST" name="logoutToApplication" finished-at="2023-07-14T15:27:54 IST" duration-ms="164" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Loggiing out ....]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutToApplication -->
        <test-method is-config="true" signature="loginToApplicationAsAdmin()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:54 IST" name="loginToApplicationAsAdmin" finished-at="2023-07-14T15:27:55 IST" duration-ms="423" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Logged in as Admin.....]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToApplicationAsAdmin -->
        <test-method retried="true" signature="reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:55 IST" name="reportsTest" finished-at="2023-07-14T15:27:55 IST" duration-ms="474" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at pgs.hms.test.ReportsPageTest.reportsTest(ReportsPageTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportsTest -->
        <test-method is-config="true" signature="logoutToApplication()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:55 IST" name="logoutToApplication" finished-at="2023-07-14T15:27:55 IST" duration-ms="127" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Loggiing out ....]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutToApplication -->
        <test-method is-config="true" signature="loginToApplicationAsAdmin()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:55 IST" name="loginToApplicationAsAdmin" finished-at="2023-07-14T15:27:56 IST" duration-ms="408" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Logged in as Admin.....]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToApplicationAsAdmin -->
        <test-method retried="true" signature="reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:56 IST" name="reportsTest" finished-at="2023-07-14T15:27:56 IST" duration-ms="499" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at pgs.hms.test.ReportsPageTest.reportsTest(ReportsPageTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportsTest -->
        <test-method is-config="true" signature="logoutToApplication()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:56 IST" name="logoutToApplication" finished-at="2023-07-14T15:27:56 IST" duration-ms="147" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Loggiing out ....]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutToApplication -->
        <test-method is-config="true" signature="loginToApplicationAsAdmin()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:56 IST" name="loginToApplicationAsAdmin" finished-at="2023-07-14T15:27:57 IST" duration-ms="466" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Logged in as Admin.....]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToApplicationAsAdmin -->
        <test-method signature="reportsTest()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:57 IST" name="reportsTest" finished-at="2023-07-14T15:27:57 IST" duration-ms="496" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at pgs.hms.test.ReportsPageTest.reportsTest(ReportsPageTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportsTest -->
        <test-method is-config="true" signature="logoutToApplication()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:58 IST" name="logoutToApplication" finished-at="2023-07-14T15:27:58 IST" duration-ms="149" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Loggiing out ....]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutToApplication -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:58 IST" name="closeBrowser" finished-at="2023-07-14T15:27:58 IST" duration-ms="180" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Browser is closing...]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="closeDB()[pri:0, instance:pgs.hms.test.ReportsPageTest@6f01b95f]" started-at="2023-07-14T15:27:58 IST" name="closeDB" finished-at="2023-07-14T15:27:58 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Data base is closing...]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeDB -->
      </class> <!-- pgs.hms.test.ReportsPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
